GamaGameHub/GamaGameHub/Extensions/GamaGameHubServiceCollectionExtension.cs

```diff
@@ -0,0 +1,15 @@
+﻿using GamaGameHub.Infrastructure.Data.Common;
+
+namespace Microsoft.Extensions.DependencyInjection
+{
+    public static class GamaGameHubServiceCollectionExtension
+    {
+        public static IServiceCollection AddApplicationServices(this IServiceCollection services)
+        {
+            services.AddScoped<IRepository, Repository>();
+            //services.AddScoped<IClientService, ClientService>();
```

Comment:
Comments should be left out of the apps code, unless they are documenting something important